bridges/snowbridge/pallets/system-frontend/src/lib.rs

```diff
@@ -198,16 +205,18 @@ pub mod pallet {
 	}
 
 	impl<T: Config> Pallet<T> {
-		pub fn send(xcm: Xcm<()>) -> DispatchResult {
-			let bridgehub = Location::new(1, [Parachain(1002)]);
-			send_xcm::<T::XcmSender>(bridgehub, xcm).map_err(|_| Error::<T>::Send)?;
+		pub fn send(origin: Location, xcm: Xcm<()>) -> DispatchResult {
+			let bridgehub = T::BridgeHub::get();
+			let (_, price) =
```

Comment:
You should also include the XCM message ID in the event, for tracing purposes

```suggestion
			let (message_id, price) =
```

Reply:
https://github.com/vgeddes/polkadot-sdk/pull/6/commits/282208d99fd8c84ef695bd258c7d05d4156c4f69

Btw:  I would assume the send hash is not the `message_id`  for tracing purpose, for doing that we need a `SetTopic` instruction.

Reply:
Nope. if the XCM message ends with a trailing topic, that topic will be used as the message_id. See

 See https://github.com/paritytech/polkadot-sdk/blob/f340e07ffd7cff69e930d19563186ff7f0fed601/cumulus/parachains/runtimes/bridge-hubs/bridge-hub-westend/src/xcm_config.rs#L242

https://paritytech.github.io/polkadot-sdk/master/staging_xcm_builder/struct.WithUniqueTopic.html

Reply:
Yeah, with the `WithUniqueTopic` router the send hash happens to be same as the `topic_id`,  but the key is  `SetTopic` instruction. On receiver end with the barrier [TrailingSetTopicAsId](https://github.com/paritytech/polkadot-sdk/blob/f340e07ffd7cff69e930d19563186ff7f0fed601/cumulus/parachains/runtimes/assets/asset-hub-westend/src/xcm_config.rs#L282) the `topic_id` is respected.

That’s basically how the indexer works under the hood.

Reply:
Anyway, the send hash seems easier to understand. Changed in 
https://github.com/vgeddes/polkadot-sdk/pull/6/commits/004c84b4cb2f3e7c0d5dc50c425ba8bd91796680